inst/templates/Makevars

```diff
@@ -8,7 +8,11 @@ all: C_clean
 $(SHLIB): $(STATLIB)
 
 $(STATLIB):
-	cargo build --lib --release --manifest-path=./rust/Cargo.toml --target-dir $(TARGET_DIR)
+	# In some environments, ~/.cargo/bin might not be included in PATH, so we need
+	# to set it here to ensure cargo can be invoked. It's added to the tail as a
```

Comment:
I'd propose to rephrase this comment a little bit.
Something like 'It is appended to PATH and therefore is only used (or searched) if cargo is absent from the user's PATH'.
Feel free to change however you like.